poetry/console/commands/run.py

```diff
@@ -28,7 +28,11 @@ def handle(self) -> Any:
         if scripts and script in scripts:
             return self.run_script(scripts[script], args)
 
-        return self.env.execute(*args)
+        try:
+            return self.env.execute(*args)
+        except FileNotFoundError:
+            self.line_error(f "<error><c1>{script}</c1>: command not found.</error>")
```

Comment:
Remove whitespace before starting the string:
```suggestion
            self.line_error(f"<error><c1>{script}</c1>: command not found.</error>")
```